src/includes/performance/traces-sampler-as-sampler/dotnet.mdx

```diff
@@ -0,0 +1,42 @@
+```java
+using Sentry;
+using Sentry.Protocol;
+
+SentrySdk.Init(options =>
+{
+    options.Dsn = "___PUBLIC_DSN___";
+
+    // To set a uniform sample rate
+    options.TracesSampleRate = 1.0;
+
+    // OR: Determine traces sample rate based on the sampling context
+    options.TracesSampler = context =>
+    {
+        var ctx = context.CustomSamplingContext;
+
+        if (ctx["url"] == "/payment")
```

Comment:
Looks like Java :trollface: 

```
return ctx[".."] switch {
 "/searh" => 0.5,
 "/bla" => 0.1
}
```
? Or better a sample that compiles with old C# versions ?

Reply:
I ported it directly from Java without thinking too much :P